config/crd/bases/osp-director.openstack.org_openstackbaremetalsets.yaml

```diff
@@ -147,9 +147,8 @@ spec:
                 secret data'
               type: string
             provisionServerName:
-              description: Name of the pre-created ProvisionServer to use. If su ProvisionServerName
-                Optional. If supplied will be used as the base Image for the baremetalset
-                instead of RhelImageURL.
+              description: ProvisionServerName Optional. If supplied will be used
+                as the base Image for the baremetalset instead of RhelImageURL.
```

Comment:
Nit: Here and elsewhere in comments, should `baremetalset`/`BaremetalSet` also be renamed?  I think it's pretty clear what this refers to in the context, so I could go either way.